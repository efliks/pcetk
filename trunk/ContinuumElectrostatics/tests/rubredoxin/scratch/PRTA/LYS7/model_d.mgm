ON_GEOM_CENT 121 2.000000
(11.583897 42.511436 -6.628633) 101 1.000000
(11.583897 42.511436 -6.628633) 101 0.500000
(11.583897 42.511436 -6.628633) 101 0.250000
