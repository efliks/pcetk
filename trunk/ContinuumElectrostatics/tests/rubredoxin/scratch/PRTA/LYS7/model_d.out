Starting /home/mikolaj/local/bin//my_2diel_solver for molecule named scratch/PRTA/LYS7//site_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:7:CE",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:7:HE1",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:7:HE2",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:7:HZ1",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:7:HZ2",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:7:HZ3",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:7:NZ",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (12.3597, 44.7491, -3.74893)) 121 2
Finer level:    (11.5839, 42.5114, -6.62863) 101 1
Finer level:    (11.5839, 42.5114, -6.62863) 101 0.5
Finest level:   (11.5839, 42.5114, -6.62863) 101 0.25


Self energy of scratch/PRTA/LYS7//site_d in two dielectric environment = 15.2395
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/LYS7//site_d with background charge set in two dielectric environment = -0.215814
(probably in kcal/mole)
Total runtime: 14.2 s user 0.11 s system 
